presto-main/src/main/java/com/facebook/presto/execution/SqlStageExecution.java

```diff
@@ -81,8 +82,11 @@
 
     private final Map<Node, Set<RemoteTask>> tasks = new ConcurrentHashMap<>();
     private final AtomicInteger nextTaskId = new AtomicInteger();
+    @GuardedBy("this")
```

Comment:
I also didn't change this in this PR, but since you asked, `nodeTaskMap` is a thread safe data structure, but the others should likely have the guarded annotation.